osmnx/bearing.py

```diff
@@ -122,27 +123,30 @@ def add_edge_bearings(G: nx.MultiDiGraph) -> nx.MultiDiGraph:
 
 
 def orientation_entropy(
-    Gu: nx.MultiGraph,
+    Gu: nx.MultiGraph | nx.MultiDiGraph,
```

Comment:
This change is unnecessary for type annotating: the `MultiDiGraph` class inherits from `MultiGraph`.

```python
import networkx as nx
G = nx.MultiDiGraph()
isinstance(G, nx.MultiGraph) # True
```

Reply:
Nice catch. FYI the `nx.MultiGraph | nx.MultiDiGraph` annotation is used in other places in case we want to update those separately.

Reply:
Thanks for flagging that. I'll review the other instances in the codebase separately so we can keep this PR clean on point.

Reply:
FYI I ended up restoring your `nx.MultiGraph | nx.MultiDiGraph` type annotations because I think they are clearer for the user even if they're redundant. That seems more important.